dapp/src/components/page/home/Profile.svelte

```diff
@@ -174,45 +161,42 @@
     </div>
   {/if}
 
-  {#if canUpload()}
-    {#if $contractAddress !== null}
-      <!-- TODO: Stylize -->
-      <span class="py-2 text-white rounded bg-green-550">
-        {'Tezos Profile deployed at '}
-        <a
-          class="text-green-900 underline"
-          target="_blank"
-          href={`https://${
-            currentNetwork
-              ? currentNetwork === 'edonet.'
-                ? 'edo2net.'
-                : `${currentNetwork}.`
-              : ''
-          }tzkt.io/${$userData.account.address}`}
-        >
-          {'tzkt.io'}
-        </a>
-      </span>
-      {#if !isAllOnChain($claimsStream)}
-        {#if isAddingClaims}
-          Adding claims....
-        {:else}
-          <PrimaryButton
-            text="Add Claims to profile"
-            class="mx-auto mt-4 bottom-6"
-            onClick={async () => {
-              await uploadNewClaim();
-            }}
-          />
-        {/if}
-      {/if}
+  {#if $contractAddress !== null}
+    <!-- TODO: Stylize -->
+    <span class="py-2 text-white rounded bg-green-550">
+      {'Tezos Profile deployed at '}
+      <a
+        class="text-green-900 underline"
+        target="_blank"
+        href={`https://${
+          currentNetwork
+            ? currentNetwork === 'edonet.'
```

Comment:
Was the full-stop intentional for this one? Any way I suppose edonet is getting old now.